src/modules/scraper/adapter/messaging/SuggestedFeesConsumer.ts

```diff
@@ -0,0 +1,73 @@
+import { OnQueueFailed, Process, Processor } from "@nestjs/bull";
+import { Logger } from "@nestjs/common";
+import { Job } from "bull";
+import { InjectRepository } from "@nestjs/typeorm";
+import { Repository } from "typeorm";
+import { DateTime } from "luxon";
+import { utils } from "ethers";
+
+import { SuggestedFeesService } from "../../../suggested-fees/service";
+import { SuggestedFeesQueueMessage, ScraperQueue } from ".";
+import { Deposit } from "../../model/deposit.entity";
+import { AppConfig } from "../../../configuration/configuration.service";
+
+@Processor(ScraperQueue.SuggestedFees)
+export class SuggestedFeesConsumer {
+  private logger = new Logger(SuggestedFeesConsumer.name);
+
+  constructor(
+    private appConfig: AppConfig,
+    private suggestedFeesService: SuggestedFeesService,
+    @InjectRepository(Deposit) private depositRepository: Repository<Deposit>,
+  ) {}
+
+  @Process()
+  private async process(job: Job<SuggestedFeesQueueMessage>) {
+    const { depositId } = job.data;
+
+    const deposit = await this.depositRepository.findOne({
+      where: { id: depositId },
+    });
+
+    if (!deposit) {
+      this.logger.verbose(`${ScraperQueue.SuggestedFees} deposit with id '${depositId}' does not exist in db`);
+      return;
+    }
+
+    if (deposit.suggestedRelayerFeePct) {
+      this.logger.verbose(
+        `${ScraperQueue.SuggestedFees} deposit with id '${depositId}' already has a suggested fees entry`,
+      );
+      return;
+    }
```

Comment:
I think we should not stop the code's execution in this case. What if we change the logic for computing the suggested fees and we want to re-run this logic for all deposits, including the ones that have this field populated?

Reply:
Good point. Will remove this if clause